site/content/docs/5.2/components/card.md

```diff
@@ -23,7 +23,9 @@ Below is an example of a basic card with mixed content and a fixed width. Cards
   {{< placeholder width="100%" height="180" class="card-img-top" text="Image cap" >}}
   <div class="card-body">
     <h5 class="card-title">Card title</h5>
-    <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+    <p class="card-text fw-bold">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+  </div>
+  <div class="card-footer mt-2">
```

Comment:
I'm not sure about this modification. In the rest of this page, `.card-footer` seems only to be used to display small grayish text really at the bottom of the card.
In Boosted, we don't have that, but for example in Bootstrap, this is really a separate area with a solid divider. If we had this divider implemented in Boosted, this button would be separated from the rest of the content.